packages/fast-usdc/src/exos/advancer.js

```diff
@@ -15,21 +15,13 @@ const { isGTE } = AmountMath;
 /**
  * @import {HostInterface} from '@agoric/async-flow';
  * @import {NatAmount} from '@agoric/ertp';
- * @import {ChainAddress, ChainHub, Denom, DenomAmount, OrchestrationAccount} from '@agoric/orchestration';
+ * @import {ChainAddress, ChainHub, Denom, OrchestrationAccount} from '@agoric/orchestration';
+ * @import {ZoeTools} from '@agoric/orchestration/src/utils/zoe-tools.js';
  * @import {VowTools} from '@agoric/vow';
  * @import {Zone} from '@agoric/zone';
  * @import {CctpTxEvidence, FeeConfig, LogFn} from '../types.js';
  * @import {StatusManager} from './status-manager.js';
- */
-
-/**
- * Expected interface from LiquidityPool
- *
- * @typedef {{
- *   lookupBalance(): NatAmount;
- *   borrow(amount: Amount<"nat">): Promise<Payment<"nat">>;
- *   repay(payments: PaymentKeywordRecord): Promise<void>
- * }} AssetManagerFacet
+ * @import {BorrowerFacet} from './liquidity-pool.js';
```

Comment:
hm... do we want to name the types of each of the facets? I can see arguments both ways, but consider...

```suggestion
 * @import {LiquidityPoolKit['borrower']} from './liquidity-pool.js';
```